docs/source/yelpsoa_configs.rst

```diff
@@ -484,7 +484,15 @@ instance MAY have:
     * ``pre_stop_command``: The command to run in your container before stopping.  This could handle gracefully stopping or checkpointing your worker, for example.
       This can be a list of strings (command + arguments) or a single string (which gets turned into a single-element list by Paasta.)
 
-    * ``termination_grace_period_seconds``: the number of seconds to allow before forcibly killing your instance.  Note that the instance will be forcibly killed after this period, so your pre_stop_command should complete well within this time period!
+    * ``pre_stop_drain_seconds``: For services registered in the mesh, we apply a default ``pre_stop_command`` which starts draining the service from the mesh and waits this many seconds before stopping the container.
```

Comment:
technically this will apply to everything, no?

(which i'd say is what we want anyway)

Reply:
Hmm. Do we actually want that for non-registered services? If we aren't triggering some sort of draining, I'm not sure there's any reason to delay the termination.

Reply:
I've updated so non-smartstack services get `sleep 0` by default (but if they specify `pre_stop_drain_seconds`, they'll get `sleep $that`).

This will cause a big bounce, but might speed up deployments for services that aren't in the mesh.